### DISCUSSION TITLE
Reply with message upon follow

### DISCUSSION THREAD

**Discussion by Bizarrebra** [Chatterino/chatterino2#2712] [Q&A] [answered]
Hi,

I see that one of the highlights mentions "Subscriptions" (I'm assuming you get the message highlighted), but is it possible to have the same for follows? Any pattern I can use for that?

Thanks in advance and keep it up!

**Comment by Mm2PL** [comment]
I don't think there is even good way to listen for follows from a chat client.

**Comment by KararTY** [comment]
This is somewhat related to #1569 (at least in the title), which was a feature request.
**I'd recommend creating a new issue requesting this feature instead of having it as a discussion, you'll get more visibility that way.**

Also for future reference, please first search the issues tab for potential requests that have been closed/are open.

**Comment by zneix** [comment, marked as answer]
> one of the highlights mentions "Subscriptions" (I'm assuming you get the **message** highlighted)

Yes, you get the **message** with user subscribing highlighted in purple, e. g.

![Don't smash it!](https://cdn.zneix.eu/GyUiIXJ.png)

It is possible to do this because we actually get the **message** whenever someone subscribes.

> is it possible to have the same for follows

However here **we do not get any message** whenever someone follows a channel. Thus, it's literally impossible to do anything about it.

#1569 is somehwat related to this but not fully. Person in that issue requested an (exportable) list of recent followers, however Bizarrebra asked for a message upon following. As I've mentioned above twitch's messaging server that we connect to doesn't reply with anything when someone follows. This would require us to implement and establish connection with twitch's websocket (or maybe pubsub, I'm not 100% sure) to get such niche thing that I doubt it's worth it really.
